public_dropin_environments/python3_keras/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This is the default base image for use with user models and workflows.
 # It contains a variety of common useful data-science packages and tools.
-FROM datarobot/python3-dropin-env-base
+FROM datarobot/dropin-env-base:debian10-py3.7-jre11
```

Comment:
I feel like this will add a lot of unused bloat for custom (training) tasks. Do you know how much storage jre adds? Does it make sense to update all public envs to have this functionality?

Reply:
https://hub.docker.com/r/datarobot/dropin-env-base/tags 
For py we've been using py3.7. slim, which was 150MB
This new image is not slim + jre, so it is 350 Mb.

As I understood, now we want all the envs to have MLOps support.